packages/ui-react/src/components/LayoutGrid/LayoutGrid.tsx

```diff
@@ -118,10 +121,10 @@ const LayoutGrid = <Item extends object>({ className, columnCount, data, renderC
             <div
               role="gridcell"
               id={`layout_grid_${rowIndex}-${columnIndex}`}
-              key={columnIndex}
+              key={getCellKey(item, rowIndex, columnIndex)}
```

Comment:
Why don't you use the required `item.mediaid` here straight?

Reply:
See https://github.com/jwplayer/ott-web-app/pull/613#discussion_r1766640006